wagtail_transfer/static_src/components/ContentImportForm/index.js

```diff
@@ -106,6 +107,8 @@ export default function ImportContentForm({
           apiBaseUrl={source.page_chooser_api}
           value={sourcePage}
           onChange={setSourcePage}
+          unchosenText="All child pages will be imported"
+          chosenText={"This page has "+(numPages-1)+"child pages."}
```

Comment:
This could be rewritten as a template string:

```javascript
chosenText={`This page has ${numPages-1} child pages.`}
```

Reply:
Good point, thanks! Changed now.